index 920b2a3..28d16c5 100644
--- a/configure.init
+++ b/configure.init
@@ -69,8 +69,8 @@ dnl determine if /dev/ptmx is available for pseudo terminals
 AC_CHECK_FILE([/dev/ptmx],
 	      [AC_DEFINE(HAVE_DEV_PTMX, 1, have /dev/ptmx)])
 
-dnl determine if /proc/self/status is available for checking TracerPid
-AC_CHECK_FILE([/proc/self/status],
+#dnl determine if /proc/self/status is available for checking TracerPid
+#AC_CHECK_FILE([/proc/self/status],
 	      [AC_DEFINE(HAVE_PROC_SELF_STATUS_FILE, 1, have /proc/self/status file)])
 
 dnl Check for pty.h

